### PR TITLE
docs: update Getting Started for React Native 0.60

### PR DESCRIPTION
**Pull request by thymikee**
RNGH works well with autolinking, so there's no need to run `link` ever again if user is on RN 0.60 or higher.

I also added a note about AndroidX and `jetifier`.

cc @satya164 @osdnk

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -80,14 +90,48 @@ public class MainActivity extends ReactActivity {
...
 ```
 
+##### Running jetifier
+
+> **Important:** This step is only necessary if you're on React Native 0.60 or greater.
+
+React Native 0.60 migrated from Support Library to AndroidX. React Native Gesture Handler is not yet compatible with AndroidX. Until then, please use [`jetifier`](https://www.npmjs.com/package/jetifier) to make it work with newest React Native:
+
+```sh
+yarn add jetifier
```
Comment (satya164) [inline docs/getting-started.md]:
```suggestion
yarn add --dev jetifier
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -80,14 +90,48 @@ public class MainActivity extends ReactActivity {
...
+```
+
+or with npm:
+
+```sh
+npm install --save-dev jetifier
+```
+
+We recommend using "postinstall" phase to run it. You can add it in your `package.json` file:
```
Comment (satya164) [inline docs/getting-started.md]: I prefer to add this step **before** running `yarn add jetifier` so it's automatically run. Or we need to include extra step `yarn jetify` after this step so the dep is jetified.

File: docs/getting-started.md [hunk 3]
```diff
@@ -80,14 +90,48 @@ public class MainActivity extends ReactActivity {
 }
 ```
 
+##### Running jetifier
+
+> **Important:** This step is only necessary if you're on React Native 0.60 or greater.
```
Comment (satya164) [inline docs/getting-started.md]: I'd rephrase it to

```suggestion
> **Important:** You only need to do this step if you're on React Native 0.60 or greater.
```

File: docs/getting-started.md [hunk 4]
```diff
@@ -10,49 +10,59 @@ The motivation for building this library was to address the performance limitati
...
 or with `npm` if you prefer:
+
 ```bash
-  npm install --save react-native-gesture-handler
+npm install --save react-native-gesture-handler
 ```
 
 #### Linking
+
+> **Important**: This step is only applicable for React Native 0.59 or lower. Since v0.60 linking happens automatically.
```
Comment (satya164) [inline docs/getting-started.md]: I'd rephrase it to

```suggestion
> **Important**: You only need to do this step if you're using React Native 0.59 or lower. Since v0.60, linking happens automatically.
```

File: docs/getting-started.md (hunk 2 above)
Comment (thymikee) [reply docs/getting-started.md]: Oh yea, good call

**Comment by thymikee** [comment]
Updated the jetifier part, as now it's running by default as a part of `run-android` command from the CLI.

**Comment by tapz** [comment]
At least for me, RNGH does not work with autolinking.

**Comment by satya164** [comment]
@tapz can you post what's not working? I have tested and it works

**Comment by thymikee** [comment]
@tapz please make sure you follow the guide completely (don't forget about manually adjusting MainActivity.java as stated there) and have Android and iOS build tools set up correctly for autolinking (I see some folks not adhering to release guides and expecting everything magically works out of the box, but we're not there yet unfortunately). 
Make sure your MainApplication.java, Gradle files and Podfile have the changes introduced in 0.60: https://react-native-community.github.io/upgrade-helper/?from=0.59.10&to=0.60.5

**Comment by tapz** [comment]
@thymikee Autolinking worker just fine with all other libraries but this.

**Comment by tapz** [comment]
@satya164 ```
/MainActivity.java:68: error: package com.swmansion.gesturehandler.react does not exist
import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
```
